contrib/views/hawq/pom.xml

```diff
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>9.1-901-1.jdbc4</version>
+            <version>${postgres.version}</version>
```

Comment:
These views are being removed in #1523, so you might want to skip the change.

Reply:
Alrighty! I know that views use their own classloader and that including an older JAR probably wasn't going to affect this fix. But I didn't want to take a chance either. Since there's a PR for their removal, I'll undo this change.